src/encoded/commands/write_submission_spreadsheets.py

```diff
@@ -147,10 +186,28 @@ def get_spreadsheets(
         request_handler: RequestHandler,
         gcc: bool = False,
         tpc: bool = False,
-        items: List[str] = None
+        items: List[str] = None,
+        example: bool = False,
     ) -> List[Spreadsheet]:
     submission_schemas = get_all_submission_schemas(request_handler)
     ordered_submission_schemas = get_ordered_submission_schemas(submission_schemas,gcc=gcc,tpc=tpc)
+    if example:
+        example_fields = get_example_fields(EXAMPLE_FILE_UUIDS,GCC_SUBMISSION_ITEMS)
+        submission_schemas = get_ordered_submission_schemas(submission_schemas,order=POPULATE_ORDER)
+        spreadsheets = []
+        for item, submission_schema in submission_schemas.items():
+            unlinked_spreadsheet = get_example_spreadsheet(
+                item,request_handler,example_fields,submission_schema
+            )
+            spreadsheet,example_fields = get_linked_spreadsheet(
+                request_handler,unlinked_spreadsheet,example_fields
+            )
+            spreadsheets.append(spreadsheet)
+        if gcc:
+            order = GCC_SUBMISSION_ITEMS
+        elif items:
+            order = items
+        return reorder_spreadsheets(spreadsheets,order)
```

Comment:
The reorder makes the order of the tabs in the spreadsheet match the GCC_SUBMISSION_ITEMS order, otherwise the sheets in the spreadsheet are in approximately reverse order